logic/src/commonTest/kotlin/com/wire/kalium/logic/data/team/TeamRepositoryTest.kt

```diff
@@ -103,4 +107,72 @@ class TeamRepositoryTest {
             assertEquals(it::class, NetworkFailure.ServerMiscommunication::class)
         }
     }
+
+    @Test
+    fun givenTeamIdAndUserDomain_whenFetchingTeamMembers_thenTeamMembersShouldBeSuccessful() = runTest {
+        val teamMember = TeamsApi.TeamMember(
+            nonQualifiedUserId = "teamMember1",
+            createdAt = "01011970",
+            legalHoldStatus = LegalHoldStatusResponse.NO_CONSENT,
+            createdBy = "nonQualiefiedUserId1",
+            permissions = TeamsApi.Permissions(copy = 1, own = 1)
+        )
```

Comment:
Would it be a good idea to extract this duplicated stub to an external class/function, so we can reuse it ?

Reply:
I was thinking about something like having another module called: test-utils and only have [Data]Generator.kt classes, where we could generate/create any type of classes we need

Reply:
@alexandreferris `test-utils` would depend on `network` to know what `TeamsApi.TeamMember` is. And `network` wouldn't be able to import `test-utils`. Only `logic` would be able to import it.

So it's as good as just extracting to another file right now.
At least we reduce redundancy inside `logic`.

Reply:
Great point @vitorhugods !

I guess then we will only be able to move forward on having the `test-utils` once we also have a specific domain model classes module.

For now I have added `TeamGenerator.kt` to create both a TeamDTO and a TeamMember (based on the scope of this ticket).